airflow/models/connection.py

```diff
@@ -156,6 +156,62 @@ class Connection(Base, LoggingMixin):
     is_extra_encrypted = Column(Boolean, unique=False, default=False)
     _extra = Column('extra', String(5000))
 
+    _types = [
+        ('docker', 'Docker Registry'),
+        ('elasticsearch', 'Elasticsearch'),
+        ('exasol', 'Exasol'),
+        ('facebook_social', 'Facebook Social'),
+        ('fs', 'File (path)'),
+        ('ftp', 'FTP'),
+        ('google_cloud_platform', 'Google Cloud Platform'),
+        ('hdfs', 'HDFS'),
+        ('http', 'HTTP'),
+        ('pig_cli', 'Pig Client Wrapper'),
+        ('hive_cli', 'Hive Client Wrapper'),
+        ('hive_metastore', 'Hive Metastore Thrift'),
+        ('hiveserver2', 'Hive Server 2 Thrift'),
+        ('jdbc', 'JDBC Connection'),
+        ('odbc', 'ODBC Connection'),
+        ('jenkins', 'Jenkins'),
+        ('mysql', 'MySQL'),
+        ('postgres', 'Postgres'),
+        ('oracle', 'Oracle'),
+        ('vertica', 'Vertica'),
+        ('presto', 'Presto'),
+        ('s3', 'S3'),
+        ('samba', 'Samba'),
+        ('sqlite', 'Sqlite'),
+        ('ssh', 'SSH'),
+        ('cloudant', 'IBM Cloudant'),
+        ('mssql', 'Microsoft SQL Server'),
+        ('mesos_framework-id', 'Mesos Framework ID'),
+        ('jira', 'JIRA'),
+        ('redis', 'Redis'),
+        ('wasb', 'Azure Blob Storage'),
+        ('databricks', 'Databricks'),
+        ('aws', 'Amazon Web Services'),
+        ('emr', 'Elastic MapReduce'),
+        ('snowflake', 'Snowflake'),
+        ('segment', 'Segment'),
+        ('sqoop', 'Sqoop'),
+        ('azure_batch', 'Azure Batch Service'),
+        ('azure_data_lake', 'Azure Data Lake'),
+        ('azure_container_instances', 'Azure Container Instances'),
+        ('azure_cosmos', 'Azure CosmosDB'),
+        ('azure_data_explorer', 'Azure Data Explorer'),
+        ('cassandra', 'Cassandra'),
+        ('qubole', 'Qubole'),
+        ('mongo', 'MongoDB'),
+        ('gcpcloudsql', 'Google Cloud SQL'),
+        ('grpc', 'GRPC Connection'),
+        ('yandexcloud', 'Yandex Cloud'),
+        ('livy', 'Apache Livy'),
+        ('tableau', 'Tableau'),
+        ('kubernetes', 'Kubernetes cluster Connection'),
+        ('spark', 'Spark'),
+        ('imap', 'IMAP')
+    ]
+
```

Comment:
Are these used anywhere? It doesn't appear to be, if so please don't add them in this PR.

Reply:
Other than this change this PR looks good to me.